tests/test_scm.py

```diff
@@ -16,6 +21,31 @@ def test_git(self):
         self.assertIsInstance(SCM(self._root_dir), Git)
 
 
+# XXX: rename Base to NoSCM
```

Comment:
Let's at least call it TestSCMBase :)

Reply:
Made this comment TODO and renamed the test case to TestNoSCM.

Reply:
Guys, I think it's way better to create an issue on Gihub instead. No one will ever go and fix this just because there is some comment with TODO. There should be some meaningful ticket with some context (while we on it it's easier to put it into some text). We can put `refactoring` or `eng` or whatever label to distinguish them.

Reply:
Do you think that this one worth an issue? I left it here mostly just because the test case class name looks strange if someone don't know that there is an intent to rename Base to NoSCM. Not sure if it is in scope of this PR, and not sure if it worth an issue.

Reply:
Issue is definitely better than just leaving a comment. It will be visible, it'll have a place to discuss it. TODOs usually stay forever as TODOs unless there is an issue :). It also extremely hard to restore the context why this TODO is in the first place. So, let's either remove it or create an issue. If @efiop thinks that it's not important, or it's obvious (like TODO: write a test) and is being tracked by other means (cov) he will just close it with "Won't fix", for example. Alternatively we might decide to merge it with some other "refactoring" issue.

Reply:
That comment doesn't make any sense now though. It is about renaming scm class, but it is in the test code :slightly_smiling_face: Let's at least remove that comment, it is not needed now. It is also not worthy of an issue. Naming is fine for the tests.

Reply:
Oh, lets then just make this empty NoSCM sugar class now. I'll be working on the explicit file abstraction and at one will move its methods tomorrow.